internal/schema2/mapped_virtual_disk_controller.go

```diff
@@ -1,16 +0,0 @@
-/*
- * HCS API
- *
- * No description provided (generated by Swagger Codegen https://github.com/swagger-api/swagger-codegen)
- *
- * API version: 2.1
- * Generated by: Swagger Codegen (https://github.com/swagger-api/swagger-codegen.git)
- */
-
-package hcsschema
-
-type MappedVirtualDiskController struct {
```

Comment:
I dont understand where this has gone? What was it replaced with that I am missing?

Reply:
See https://github.com/Microsoft/hcsshim/pull/258#discussion_r203079897